src/js/InfographicTemplates/ADHDTemplateOne.js

```diff
@@ -0,0 +1,372 @@
+                height: 160,
+                fill: '#ffd34e',
+                group: chartBackgroundGroup,
+            }
+        );
+        this._graphicsHandler.AddGraphic({
+            type: 'header',
+            graphic: chartBackground,
+            group: chartBackgroundGroup
+        });
+
+        var waffleOneGroup = new Konva.Group({
+            name: 'WaffleChart',
+            id: 0,
+            offsetY: -90,
+            offsetX: -55,
+            width: 280,
+            height: 100,
+        });
+        
+        sectionOne.add(waffleOneGroup);
+        
+        var ICON_FONT = '"Font Awesome 5 Free"';
+        console.log("Default offset: " + DEFAULT_OFFSET);
+        var ADHDPreset = GenerateWafflePreset(ADHD, DARK_BLUE, DEFAULT_OFFSET, ICON_FONT),
+            redADHDPreset = GenerateWafflePreset(ADHD, RED, DEFAULT_OFFSET - 1, ICON_FONT);
+
+        var waffleOne = new WaffleChart({
+            numerator: 9, 
+            denominator: 10, 
+            group: waffleOneGroup, 
+            presetA: ADHDPreset, 
+            presetB: redADHDPreset, 
+            fontSize: 20,
+            maxIconsPerRow: 10
+        });
+        this._chartHandler.AddChart({
+            chart: waffleOne, 
+            group: waffleOneGroup, 
+            type:'Waffle'
+        });
+        this._chartHandler.GetChart(this._chartHandler.GetCurrChartID()).CreateChart();
+
+        var waffleOneTextDiv = document.createElement('div')
+        var waffleOneText = '<p style="margin: 0px;"><span style="font-size: 20px; font-family: Roboto, sans-serif; font-weight: 400; line-height: 1.0;">'
+        + '8.4% of children  ages 3-17 are</span></p>'
+        +'<p style="margin: 0px;"><span style="font-size: 20px; font-family: Roboto, sans-serif; font-weight: 400; line-height: 1.0;">'
+        + 'diagnosed with ADHD</span></p>';
+        waffleOneTextDiv.innerHTML = waffleOneText;
+        waffleOneTextDiv.style.color = '#ae4d34';
+        this._textHandler.AddTextElem({
+            textElem: waffleOneTextDiv, 
+            group: sectionOne, 
+            x: 50, 
+            y: 175
+        });
+        this._textHandler.SetCSSInfo({
+            id: this._textHandler.GetCurrID(),
+            fontFamily: roboto400,
+            fontSize: '20px',
+            textColor: waffleOneTextDiv.style.color,
+            lineHeight: '1.0',
+        });
+        this._textHandler.SetCSSInfo({
+            id: this._textHandler.GetCurrID(),
+            fontFamily: roboto400,
+            fontSize: '20px',
+            textColor: waffleOneTextDiv.style.color,
+            lineHeight: '1.0',
+        });
+
+        var waffleTwoGroup = new Konva.Group({
+            name: 'WaffleChart',
+            id: 0,
+            offsetY: -75,
+            offsetX: -500,
+            width: 180,
+            height: 100,
+        });
+        sectionOne.add(waffleTwoGroup);
+
+        var BoysPreset = GenerateWafflePreset(ADHD, DARK_BLUE, DEFAULT_OFFSET, ICON_FONT),
```

Comment:
Formatting: variable names should start lower case

Also, you should use let instead of var